account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/AuthenticateHandler.java

```diff
@@ -56,7 +58,12 @@ public APIGatewayProxyResponseEvent handleRequest(
         return isWarming(input)
                 .orElseGet(
                         () -> {
-                            LOGGER.info("Request received to the AuthenticateHandler");
+                            String sessionId =
+                                    RequestHeaderHelper.getHeaderValueOrElse(
+                                            input.getHeaders(), SESSION_ID_HEADER, "");
+                            LOGGER.info(
```

Comment:
To save you from appending it to each log line, you could use the `attachSessionIdToLogs`.

Reply:
Looks like a good idea, don't have the actual session in AM, but could add a similar method that just adds the session id?